### PR TITLE
Do not hide not secret settings in the web plugin UI

### PR DESCRIPTION
**Pull request by alexintech**
# What this PR does

In the web plugin UI Settings -> Env Variables if variable is not secret, do not hide its value.

## Checklist

- [ ] Unit, integration, and e2e (if applicable) tests updated
- [ ] Documentation added (or `pr:no public docs` PR label added if not required)
- [x] `CHANGELOG.md` updated (or `pr:no changelog` PR label added if not required)

### CODE REVIEW

**Comment by Konstantinov-Innokentii** [comment]
@vadimkerr what do you think?

**Comment by vadimkerr** [comment]
I personally like all values hidden by default so it's safe to open this page when screen sharing / making screenshots. Hiding all values by default also makes things simpler IMO, so there's only two options – either all settings are hidden or all settings are exposed.
wdyt @grafana/grafana-oncall-frontend @Matvey-Kuk?

**Comment by Konstantinov-Innokentii** [comment]
@vadimkerr I don't like, that all values are hidden. With ability to change phone provider or ESP for inbound email it feels strange to hide chosen options.

**Comment by alexintech** [comment]
When I want to change the value of a setting, I scroll down for some non-secret setting. But when I press the "Edit" button, the input is cleared, so you can't see the current value, because of this `<Text>` behavior.

https://github.com/grafana/oncall/blob/bd142927b590f8b8ea2ec63efe1ac1155a6fce3a/grafana-plugin/src/components/Text/Text.tsx#L62

I close the modal window, scroll up, click the "Show values" button, scroll down again, and only then can I edit the value. After refreshing the page, all the values are hidden again. That's a little cumbersome.

File: CHANGELOG.md [hunk 1]
```diff
@@ -35,6 +41,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 
 - Changed mobile notification title and subtitle. Removed the body. by @imtoori [#2027](https://github.com/grafana/oncall/pull/2027)
+- Do not hide not secret settings in the web plugin UI by @alexintech ([#1964](https://github.com/grafana/oncall/pull/1964))
```
Comment (joeyorlando) [inline CHANGELOG.md:44]:
```suggestion
```

File: grafana-plugin/src/pages/settings/tabs/LiveSettings/LiveSettingsPage.tsx
Comment (joeyorlando) [inline grafana-plugin/src/pages/settings/tabs/LiveSettings/LiveSettingsPage.tsx:1]: _nit_ can the `hideValues && item.is_secret` logic be deduplicated into a function on the `LiveSettings` class?

Comment (joeyorlando) [inline grafana-plugin/src/pages/settings/tabs/LiveSettings/LiveSettingsPage.tsx:1]: _nit_ can the `hideValues && item.is_secret` logic be deduplicated into a function on the `LiveSettings` class?

**Review by joeyorlando — APPROVED** [review]
Overall LGTM. Consider adding a jest snapshot test for the two cases where a value is hidden, and when it is not hidden.

**Comment by alexintech** [comment]
> Overall LGTM. Consider adding a jest snapshot test for the two cases where a value is hidden, and when it is not hidden.

I've tried to write a test, but I have not much experience in frontend. All my attempts failed.

File: grafana-plugin/src/pages/settings/tabs/LiveSettings/LiveSettingsPage.tsx
Comment (alexintech) [reply grafana-plugin/src/pages/settings/tabs/LiveSettings/LiveSettingsPage.tsx:1]: I don't suppose that it's necessary.

**Pull request merged**